layer1/monitor/monitor.go

```diff
@@ -191,7 +191,7 @@ func (mon *monitor) Start() error {
 
 // eventLoop to process the events and chain changes.
 func (mon *monitor) eventLoop(logger *logrus.Entry) {
-	gcTimer := time.After(constants.MonDBGCFreq)
+	gcTimer := time.After(time.Second * constants.MonDBGCFreq)
```

Comment:
A constant can include `time.Second`, doesn't need to be specified everywhere it's used, e.g.:

```go
const freq = 30 * time.Second
```

Reply:
I agree, will be changing this!